### ISSUE TITLE
Unlink/Remove Link option does not activated when there are multiple links in selected texts.

### ISSUE THREAD

**Issue by mostafa-K-raihan** [tinymce/tinymce#5599]
**What is the current behavior? Describe the bug**
If the editor contains multiple links inside a large texts, selecting all texts does not activate the remove link/unlink button. It only gets activated when a single link is selected. This feature was there in version < 5. We recently migrated to version 5 and lost the feature.

**Please provide the steps to reproduce and if possible a minimal demo of the problem via [fiddle.tinymce.com](http://fiddle.tinymce.com/) or similar.**
- write a large text with multiple links
- select all text using ctrl + a
- remove link / unlink will not be activated 

**What is the expected behavior?**
- remove link / unlink should be activated and work accordingly

**Which versions of TinyMCE, and which browser / OS are affected by this issue? Did this work in previous versions of TinyMCE?**
tried with tinymce v5, chrome 81 on macOS (Mojave)

**Comment by EkimChau** [comment]
Hi mostafa-K-raihan ,

"selecting all texts does not activate the remove link/unlink button" Do you mean the link button in the main toolbar? or the context menu? 

I can't replicate the issue you are reporting as both TinyMCE 4 and TinyMCE 5 appear to behave the same way when i tested this. 

Could you make a [fiddle](fiddle.tinymce.com) or screenshots of the replication for TinyMCE 4 and TinyMCE 5 that shows the problem?

**Comment by mostafa-K-raihan** [comment]
Hi @EkimChau 
```
<script type="text/javascript">
tinymce.init({
    selector: "textarea",
    plugins: [
        "link"
    ],
    toolbar: "link unlink"
});
</script>

<form method="post" action="dump.php">
    <textarea name="content"></textarea>
</form>
```
try this in both version 4 and 5, have multiple links in the text area. select all the text, you will notice the difference. lemme know if you need anything else !

**Comment by EkimChau** [comment]
Hi mostafa-K-raihan,

I was able to replicate the issue you were seeing with the provided config.  This is a regression and we have now added this to our bug backlog to be fixed.

Thank you for reporting the issue

**Comment by mostafa-K-raihan** [comment]
@EkimChau thank you for your response. Any idea, how long do we have to wait for the fix ?

**Comment by lnewson** [comment]
This issue has been fixed in TinyMCE 5.5.0, which was released to the community earlier today. Thanks again for reporting this and please let us know if you have any further issues.
